x-pack/plugins/actions/server/builtin_action_types/email.ts

```diff
@@ -58,7 +78,8 @@ function validateConfig(
   // translate messages.
   if (config.service === JSON_TRANSPORT_SERVICE) {
```

Comment:
Not sure if this check makes any sense here with the current changes `if (config.service === JSON_TRANSPORT_SERVICE) {`
What it supposed to do? Is it possible to have `JSON_TRANSPORT_SERVICE` as a value for `config.service` now?

Reply:
It looks like it's used specifically for functional tests.